arcus/azureml/projectutils.py

```diff
@@ -1,16 +1,22 @@
+'''
+The projectutils module provides common operations related to data/ai projects
+'''
+
 import os
 
 def initProjectStructure():
```

Comment:
please use the standard python naming conventions: so, better having init_project_structure
I would also give an optional argument to the function call that has a different folder name:
`def init_project_structure(root_folder: str = None)`

Reply:
forgot to link to the standard namings (pep8): https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments